build-system/server/app.js

```diff
@@ -900,7 +900,11 @@ app.get(['/dist/v0/amp-*.js'], (req, res, next) => {
 app.get('/test/manual/amp-video.amp.html', runVideoTestBench);
 
 app.get(
-  ['/examples/*.html', '/test/manual/*.html', '/test/fixtures/e2e/*/*.html'],
+  [
+    '/examples/(**/)?*.html',
+    '/test/manual/(**/)?*.html',
+    '/test/fixtures/e2e/*/*.html',
```

Comment:
Let me know if we want to proxy files from e2e's nested folder  : )

Reply:
Yeah sure! :)

Reply:
done : ) now we also support files directly under e2e folder, but I believe that's fine.